wire-ios/WireUITests/ApiClients/BackendClient.swift

```diff
@@ -77,17 +75,42 @@ enum BackendClient {
 
         let pureResponse = response as! HTTPURLResponse
         if pureResponse.statusCode != 200 {
-            print("Error! got status code \(pureResponse.statusCode)")
-            print("Response: \(pureResponse.description)")
+
             throw RuntimeError("Error \(pureResponse.description)")
+
         }
 
         let message: ActivationCodeReponse = try JSONDecoder().decode(ActivationCodeReponse.self, from: responseData)
         return (message.code, message.key)
     }
 
+    static func sendVerificationCode(email: String, password: String) async throws {
+        let access_token = try await loginViaAPI(email: email, password: password)
+        let body: [String: Any] = [
+            "action": "delete_team",
+            "email": email
+        ]
+
+        let url = URL(string: "\(backendURL)/v8/verification-code/send")
```

Comment:
suggestion: pass the apiVersion 
```suggestion
        let url = URL(string: "\(backendURL)/v8/verification-code/send")
```

Reply:
question: should it move to wireNetwork?

Reply:
parameterised with apiVersion